package.json

```diff
@@ -42,6 +42,8 @@
   },
   "devDependencies": {
     "fs-extra": "^0.16.5",
+    "grunt-banner": "^0.6.0",
```

Comment:
Is this dependency a result of an old master branch? Or did you manually add this line and the next back in?